lib/internal/bootstrap_node.js

```diff
@@ -196,7 +200,12 @@
       enumerable: false,
       configurable: true
     });
-    global.process = process;
+    Object.defineProperty(global, 'process', {
```

Comment:
Are there many? https://nodejs.org/api/globals.html documents only a bit more than a dozen "globals", and half of them aren't global, they are module-scoped variables. Perhaps all the "globals" should be module-scoped variables? So that its impossible to mess with them globally?